src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -184,7 +189,28 @@ impl View {
         scene.views.main.remove(&shape_system.shape_system.symbol);
         scene.views.viz.add(&shape_system.shape_system.symbol);
 
-        Self {logger,display_object,background,overlay}
+        let shape_system = scene.shapes.shape_system(PhantomData::<overlay::Shape>);
+        scene.views.main.remove(&shape_system.shape_system.symbol);
+        scene.views.viz.add(&shape_system.shape_system.symbol);
+
+        let div            = web::create_div();
+        let background_dom = DomSymbol::new(&div);
+        // TODO : We added a HTML background to the `View`, because native background was overlapping
+        //        the JS visualization. This should be further investigated while fixing rust
+        //        visualization displaying. (#796)
+        background_dom.dom().set_style_or_warn("width"         ,"0"        ,&logger);
+        background_dom.dom().set_style_or_warn("height"        ,"0"        ,&logger);
+        background_dom.dom().set_style_or_warn("z-index"       ,"1"        ,&logger);
```

Comment:
Why we set  "z-index" to 1? It should be rather value less than 0 if it should be below other visualization elements? And I have impression that setting this property is not needed at all.

Reply:
It is necessary as I have explained on DM. TLDR: It works properly only if Z index is 1, if 0 it didnt work, and the visualization class - so every JS visualization container has Z-index of 2, and to end - the rust content is always on top of js visualization as it is in back layer.